docs/tutorial.md

```diff
@@ -204,7 +204,7 @@ kubectl create secret docker-registry regcred \
                     --docker-email=<your-email>
 ```
 
-To be able to use this `secret` the `TaskRun` needs to use a `ServiceAccount`.
+To be able to use this `secret`, the `TaskRun` needs to use a `ServiceAccount`.
```

Comment:
The comma is unnecessary since both clauses contain a verb. A comma is only needed to separate a clause without a verb that prefixes the main clause, such as "In the meantime, we install Tekton."